akd_core/benches/parallel_vrfs.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+//
+// This source code is licensed under both the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree and the Apache
+// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+// of this source tree.
+
+//! Benchmarks for parallel vs sequential VRF calculations
+
+extern crate criterion;
+use self::criterion::*;
+use akd_core::{ecvrf::VRFKeyStorage, AkdLabel};
+
+fn bench_parallel_vrfs(c: &mut Criterion) {
+    // utilize all cores available
+    let runtime = tokio::runtime::Builder::new_multi_thread().build().unwrap();
+    let limited_runtime = tokio::runtime::Builder::new_multi_thread()
+        .worker_threads(4)
+        .build()
+        .unwrap();
+
+    // generate 1K labels to do VRFs for
+    let labels = (0..1_000)
+        .into_iter()
+        .map(|i| {
+            let name = format!("user {}", i);
+            (AkdLabel::from_utf8_str(&name), false, i as u64)
+        })
+        .collect::<Vec<_>>();
+    let labels_clone = labels.clone();
+
+    c.bench_function("Sequential VRFs", |b| {
+        b.iter(|| {
+            let key = runtime
```

Comment:
yeah it's part of the proper benchmark, because the parallel will do this op too. If we don't it's not a "true' comparison, but I see what you're getting at. But I think it should be measured.